proj4/src/main/scala/geotrellis/proj4/ConusAlbers.scala

```diff
@@ -0,0 +1,7 @@
+package geotrellis.proj4
+
+object ConusAlbers extends CRS {
+  lazy val proj4jCrs = factory.createFromName("EPSG:5070")
+
+  def epsgCode: Option[Int] = CRS.getEPSGCode(toProj4String + " <>")
```

Comment:
This is a weird way to call this...I know this comes from the other defined CRS's, but shouldn't they all be something like:

```scala
def epsgCode: Option[Int] = Some(5070)
```

Can you change this and the others to do that?